src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -637,7 +646,7 @@ internal async Task<Collection<FunctionDescriptor>> GetFunctionDescriptorsAsync(
             Collection<FunctionDescriptor> functionDescriptors = new Collection<FunctionDescriptor>();
             var httpFunctions = new Dictionary<string, HttpTriggerAttribute>();
 
-            if (!_environment.IsPlaceholderModeEnabled())
+            if (!_environment.IsPlaceholderModeEnabled() && _httpInvokerOptions == null)
```

Comment:
you will always get default value for _httpInvokerOptions. This will never be null.

Reply:
you are right. Will fix it. Thanks!